play-json/shared/src/test/scala/play/api/libs/json/JsonSharedSpec.scala

```diff
@@ -270,7 +270,7 @@ class JsonSharedSpec extends WordSpec
         "key3" -> js.arr(1, "tutu")
       )
 
-      js.prettyPrint(jo) mustEqual ("""{
+      js.prettyPrint(jo).replaceAllLiterally("\r\n", "\n") mustEqual ("""{
```

Comment:
Why is this necessary now?

Reply:
otherwise the test breaks on windows

Reply:
Oh, okay. Thanks for fixing that too. :-)